README.md

```diff
@@ -55,35 +61,35 @@ $ cat example/config.yml
     count: $.count # dynamic value
     boolean: $.some_boolean
 
+headers:
+  X-Dummy: my-test-header
+
 $ python -m SimpleHTTPServer 8000 &
 Serving HTTP on 0.0.0.0 port 8000 ...
 
-$ ./json_exporter http://localhost:8000/example/data.json example/config.yml &
-INFO[2016-02-08T22:44:38+09:00] metric registered;name:<example_global_value>
-INFO[2016-02-08T22:44:38+09:00] metric registered;name:<example_value_boolean>
-INFO[2016-02-08T22:44:38+09:00] metric registered;name:<example_value_active>
-INFO[2016-02-08T22:44:38+09:00] metric registered;name:<example_value_count>
-127.0.0.1 - - [08/Feb/2016 22:44:38] "GET /example/data.json HTTP/1.1" 200 -
+$ ./json_exporter --config.file examples/config.yml &
 
-
-$ curl http://localhost:7979/metrics | grep ^example
-example_global_value{environment="beta"} 1234
+$ curl "http://localhost:7979/probe?target=http://localhost:8000/examples/data.json" | grep ^example
```

Comment:
Looks like that anybody can set up http server, specify its address after ?target= and using tcpdump/base64 -d get credentials from the config.yml